### PR TITLE
Removed identity as a separate data property of entities.

### PR DESCRIPTION
**Pull request by jesper-friis**
Entities are uniquely identified by their IRI. Hence, there are no need for the hasIdentity data property.

This PR replaces #6. It removed identity as a separate data property of entity.

### CODE REVIEW

**Comment by francescalb** [comment]
I think this makes sense. But doesn't this affect the Dlite datamodel that we use?

**Comment by jesper-friis** [comment]
> I think this makes sense. But doesn't this affect the Dlite datamodel that we use?

No. The json representation will be unaffected. The namespace/version/name identifier for a metadata will correspond to its ontological IRI.

File: README.md [hunk 1]
```diff
@@ -21,7 +21,6 @@ The root concept in the taxonomy of this ontology is called **DataModel**.
 
 ### Entity
 The **Entity** is the most central concept in this ontology.  It is the class of individuals that represent any self-contained piece of information.  In addition has it the following parts (composition):
```
Comment (francescalb) [inline README.md]:
```suggestion
The **Entity** is the most central concept in this ontology.  It is the class of individuals that represent any self-contained piece of information.  
It id uniquely defined by its IRI. In addition it has the following parts (composition):
```

File: datamodel.ttl [hunk 2]
```diff
@@ -11,34 +11,48 @@
...
 ###  http://emmo.info/datamodel#aggregation
-:aggregation rdfs:subPropertyOf emmo:EMMO_6703954e_34c4_4a15_a9e7_f313760ae1a8 .
+:aggregation rdfs:subPropertyOf emmo:EMMO_8785be5a_2493_4b12_8f39_31907ab11748 .
 
 
 ###  http://emmo.info/datamodel#composition
 :composition rdfs:subPropertyOf emmo:EMMO_d893d373_b579_4867_841e_1c2b31a8d2c6 .
 
 
-###  http://emmo.info/datamodel#dataModelRelation
```
Comment (francescalb) [inline datamodel.ttl:38]: Why is dataModelRelation removed? Whjat was it used for before?

Comment (jesper-friis) [reply datamodel.ttl:38]: > Why is dataModelRelation removed? Whjat was it used for before?

It was removed because it wasn't used before.

**Pull request merged**